cirq/google/engine/engine.py

```diff
@@ -61,6 +62,20 @@ class ProtoVersion(enum.Enum):
 Program = Union[circuits.Circuit, Schedule]
 
 
+def user_project_header_request_builder(project_id):
+    """Provides a request builder that sets a user project header on engine
+    requests to allow using standard OAuth credentials.
+    """
+
+    def request_builder(*args, **kwargs):
+        from apiclient import http as apiclient_http
```

Comment:
is there a reason this import is here?  If so document, if not move up with other imports

Reply:
Moved up.